proto/osmosis/streamswap/v1/genesis.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package osmosis.streamswap.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "osmosis/streamswap/v1/state.proto";
+import "osmosis/streamswap/v1/params.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v10/x/streamswap/types";
+
+// GenesisState defines the streamswap module's genesis state.
+message GenesisState {
+  repeated Sale sales = 1 [ (gogoproto.nullable) = false ];
+  repeated UserPositionKV user_positions = 2 [ (gogoproto.nullable) = false ];
+  uint64 next_sale_id = 3;
+  Params params = 4 [ (gogoproto.nullable) = false ];
+}
+
+message UserPositionKV {
+  // user account address
+  string acc_address = 1;
+  uint64 sale_id = 2;
+  UserPosition u = 3 [ (gogoproto.nullable) = false ];
```

Comment:
please consider a more descriptive name for `u`

Reply:
I wanted something short - type already provides everything. `user_position` would be long.

Reply:
ok, changed to `user_position`